sdk/tests/conformance2/textures/misc/tex-mipmap-levels.html

```diff
@@ -122,6 +122,20 @@
   wtu.clearAndDrawUnitQuad(gl);
   wtu.glErrorShouldBe(gl, gl.NO_ERROR, "clearAndDrawQuad should succeed");
   wtu.checkCanvas(gl, [0, 0, 0, 255], "incomplete; should draw with [0, 0, 0, 255]");
+
+  // Test base level texture isn't specified.
+  var tex = gl.createTexture()
+  gl.bindTexture(gl.TEXTURE_2D, tex);
+  wtu.fillTexture(gl, tex, 8, 8, [255, 0, 0, 255], 2, gl.RGBA, gl.UNSIGNED_BYTE);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "fillTexture(8,8,2) should succeed");
+  wtu.fillTexture(gl, tex, 4, 4, [255, 0, 0, 255], 3, gl.RGBA, gl.UNSIGNED_BYTE);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "fillTexture(4,4,3) should succeed");
+  wtu.fillTexture(gl, tex, 2, 2, [0, 0, 255, 255], 4, gl.RGBA, gl.UNSIGNED_BYTE);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "fillTexture(2,2,4) should succeed");
```

Comment:
Looking above, it seems to refer to arguments 2, 3, and 5 above. (2x2 texture at level 4.) Is that correct?